### DISCUSSION TITLE
The animation will out of box

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#9290] [Q&A] [answered]
**Describe the bug**
Animation will make the control render in outside.

**To Reproduce**
```C#
private PageSlide slide = new(TimeSpan.FromMilliseconds(5000));

....

await slide.Start(content1, content2, now < Tabs.SelectedIndex, CancellationToken.None);
```

![image](https://user-images.githubusercontent.com/26276037/198046849-267c74f6-0657-4e3e-a518-db1ae10c60df.png)
![image](https://user-images.githubusercontent.com/26276037/198047003-9f57dc7a-a562-48f2-9858-2149dd08085d.png)
![image](https://user-images.githubusercontent.com/26276037/198047152-14caf8b8-1b9d-41ef-9bd5-edfb6e2610e1.png)
![image](https://user-images.githubusercontent.com/26276037/198047287-7afbcf69-819c-404d-b596-2c643448055f.png)

![GIF](https://user-images.githubusercontent.com/26276037/198047879-c6278f43-7b8b-40d1-80d0-89bd4d1cf756.gif)

**Desktop (please complete the following information):**

- OS: Windows
- Version 0.10.18

**Comment by timunie** [comment]
Cam you try to set ClipRoBounds to true to your animated control?

**Comment by Coloryr** [comment]
> Cam you try to set ClipRoBounds to true to your animated control?

Did you mean control's attribute `ClipRoBounds `?

```
<Grid
                Name="Tab1"
                Margin="10,50,10,10"
                ClipRoBounds="true" />
```

```
System.Xaml.XamlException: “Unable to resolve suitable regular or attached property ClipRoBounds on type Avalonia.Controls:Avalonia.Controls.Grid Line 30, position 14.”
```

It does not exist

https://github.com/AvaloniaUI/Avalonia/search?q=ClipRoBounds

**Comment by timunie** [comment, marked as answer]
Sorry I had a typo. It's: `ClipToBounds`

**Comment by Coloryr** [comment]
> Sorry I had a typo. It's: `ClipToBounds`

Thanks

**Reply by timunie** [reply]
I converted this issue into a Q&A. That way others may find this helpful.
